source/core/material/warp.cpp

```diff
@@ -59,6 +59,13 @@ namespace pov
 
 const DBL COORDINATE_LIMIT = 1.0e17;
 
+enum BlackHoleType
+{
+    kBlackHoleSpherical_Original=0,
+    kBlackHoleSpherical_Clamped,
+    kBlackHoleSpherical_WhatMightBeUseful
```

Comment:
For clarity, I specifically meant the `kBlackHoleSpherical_WhatMightBeUseful` value. It's not used, as far as I can see, so it shouldn't be there. The others correspond to actual implementations, so I guess they're ok.